src/components/package-show/package-show.js

```diff
@@ -13,13 +13,15 @@ import List from '../list';
 import TitleListItem from '../title-list-item';
 import ToggleSwitch from '../toggle-switch';
 import Modal from '../modal';
+import PackageCustomCoverage from '../package-custom-coverage/';
 import { formatISODateWithoutTime } from '../utilities';
 import styles from './package-show.css';
 
 export default class PackageShow extends Component {
   static propTypes = {
     model: PropTypes.object.isRequired,
-    toggleSelected: PropTypes.func.isRequired
+    toggleSelected: PropTypes.func.isRequired,
+    customCoverageSubmitted: PropTypes.func.isRequired
```

Comment:
as we add more of these maybe we should bundle the field-specific edits into a single PUT with only the changed info.  the backend is smart enough to collapse nil values into the model's pre-existing values.  i dunno how this jives with the redux data layer though

Reply:
What do you mean by field specific edits?

Reply:
like rather than have `toggleSelected` and `customCoverageSubmitted` maybe just `packageEdited`.  collect the attributes that have been touched, construct the payload and fire it off to the server.  now that i say it out loud i'm not as sure about it.  i'm just thinking long term all these edits are doing the same thing - sending the full payload to the server with one thing changed and the other things nullified.  could be an opportunity here to DRY it up, but now i'm not so sure that DRYing wouldn't be at the cost of readability

Reply:
aaaah i see. Then that `packageEdited` function will almost be like a validation function in a sense...That could get crazy really fast with lots of logic in that function rather than have explicit calls to function. But i can see the argument for the one function if its not too long.